cc-ui/src/components/ProtectedRoute.tsx

```diff
@@ -1,16 +1,16 @@
-import React, { ReactElement } from 'react';
-import { Navigate } from 'react-router-dom';
-import { ReactChildrenElement } from '../types/Reactchildren';
-import { useAuth } from './AuthProvider';
-
-export const ProtectedRoute: React.FC<ReactChildrenElement> = ({
-  children,
-}) => {
-  const { user } = useAuth();
-
-  if (!user) {
-    return <Navigate to='/' />;
-  }
-
-  return children;
-};
+import React, { ReactElement } from 'react';

```

Comment:
what has changed here?

Reply:
hmm, not sure why this is here, there shouldn't be any changes